contributors.yaml

```diff
@@ -0,0 +1,277 @@
+- given-names: Timothy
+  family-names: Hallett
+  orcid: "https://orcid.org/0000-0002-9681-0424"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/timothy.hallett"
+  github-username: tbhallett
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Andrew
+  family-names: Phillips
+  orcid: "https://orcid.org/0000-0003-2384-4807"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/5430"
+  github-username: andrew-phillips-1
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Asif
+  family-names: Tamuri
+  orcid: "https://orcid.org/0000-0001-6633-3789"
+  affiliation: "University College London"
+  website: "https://www.homepages.ucl.ac.uk/~ucbtaut/"
+  github-username: tamuri
+  role: "Lead developer"
+  contributions:
+    - Software development
+- given-names: Joseph
+  family-names: Mfutso-Bengo
+  orcid: "https://orcid.org/0000-0001-9396-3700"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://sphfm.medcol.mw/staff/dr-joseph-mfutso-bengo-phd" # to check - link seems not to work
+  role: "Lead policy translation"
+  contributions:
+    - Policy translation
+- given-names: Tim
+  family-names: Colbourn
+  orcid: "https://orcid.org/0000-0002-6917-6552"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/3084"
+  github-username: timcolbourn
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Joseph
+  family-names: Collins
+  orcid: "https://orcid.org/0000-0003-4801-7218"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/70734"
+  github-username: joehcollins
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Iwona
+  family-names: Hawryluk
+  orcid: "https://orcid.org/0000-0001-6919-4366"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/i.hawryluk19"
+  github-username: ihwaryluk
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Britta
+  family-names: Jewell
+  orcid: "https://orcid.org/0000-0003-4275-8509"  # to check
+  affiliation: "Imperial College London" # to check
```

Comment:
@brittajewell - not sure if you are still getting GitHub notifications but if so can I check if this is a valid ORCID for you and if the affiliation should be updated?